news/2017-08-15-osu-mania-4k-world-cup-group-stage-recap.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "osu!mania 4K World Cup - Group Stage Recap"
+date: 2017-08-15 14:37:30 +0000
+---
+
+Hi everyone, mangomizer here! Summer may be ending, but the excitement continues! I’m here today of course, to catch up on the latest events regarding the hugely anticipated osu!mania 4K World Cup!
```

Comment:
drop "hi everyone mangomizer here" line from leading paragraph, rephrase to something more passive and focused on osu!mania 4K. maybe briefly reference (1 line) the super exciting stuff, these paragraphs are supposed to bait people into reading them, not explain who's doing the article and what its for